apache/superset
# feat(HelmChart): Add securityContext on pod level

### SUMMARY
Follow up on https://github.com/apache/superset/pull/20509
It is also worth add securityContext configuration on a container level.

### TESTING INSTRUCTIONS
In order to test changes, one needs to change securityContext specification in values.yaml for any Container and check if after deployment it changed desired outcome in deployed Container.

### ADDITIONAL INFORMATION


- [x] Has associated issue: https://github.com/apache/superset/issues/20489
